lib/__tests__/disableRanges-integration.test.mjs

```diff
@@ -1,9 +1,7 @@
-'use strict';
-
-const blockNoEmpty = require('../rules/block-no-empty');
-const maxLineLength = require('../rules/max-line-length');
-const selectorCombinatorSpaceBefore = require('../rules/selector-combinator-space-before');
-const stringQuotes = require('../rules/string-quotes');
+import blockNoEmpty from '../rules/block-no-empty/index.js';
+import maxLineLength from '../rules/max-line-length/index.js';
+import selectorCombinatorSpaceBefore from '../rules/selector-combinator-space-before/index.js';
+import stringQuotes from '../rules/string-quotes/index.js';
```

Comment:
You could maybe import them all from `stylelint/lib/rules
/index.js`. But it would require to rename them all due to the hyphens.
Your choice.